pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/Constants.java

```diff
@@ -32,4 +32,5 @@ private Constants() {
   public static final String THETA_TUPLE_SKETCH_NOMINAL_ENTRIES = "nominalEntries";
   public static final String PERCENTILETDIGEST_COMPRESSION_FACTOR_KEY = "compressionFactor";
   public static final String SUMPRECISION_PRECISION_KEY = "precision";
+  public static final String KLL_DOUBLE_SKETCH_K_VALUE = "kValue";
```

Comment:
Maybe we can simply call this `K` as that's what DataSketches seems to use - https://datasketches.apache.org/docs/KLL/KLLAccuracyAndSize.html#:~:?